src/shared/bg_pmove.cpp

```diff
@@ -4605,15 +4588,7 @@ void PmoveSingle( pmove_t *pmove )
 
 	// determine the time
 	pml.msec = pmove->cmd.serverTime - pm->ps->commandTime;
-
-	if ( pml.msec < 1 )
-	{
-		pml.msec = 1;
-	}
-	else if ( pml.msec > 200 )
-	{
-		pml.msec = 200;
-	}
+	Math::Clamp( pml.msec, 1, 200 );
```

Comment:
This does nothing. use this instead:
```suggestion
 	pml.msec = Math::Clamp( pml.msec, 1, 200 );
```

Reply:
always confused by that math lib... thanks